hack/local-up-cluster.sh

```diff
@@ -221,6 +221,7 @@ function start_apiserver {
       --port="${API_PORT}" \
       --etcd_servers="http://127.0.0.1:4001" \
       --service-cluster-ip-range="10.0.0.0/24" \
+      --runtime-config=experimental/v1=true \
```

Comment:
For local-up-cluster, why would we not want experimental enabled?  It's
only a dev cluster.

On Fri, Aug 7, 2015 at 5:34 AM, Marcin Wielgus <notifications@github.com>
wrote:

> In hack/local-up-cluster.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12217#discussion_r36504307>
> :
>
> > @@ -221,6 +221,7 @@ function start_apiserver {
> >        --port="${API_PORT}" \
> >        --etcd_servers="http://127.0.0.1:4001" \
> >        --service-cluster-ip-range="10.0.0.0/24" \
> > +      --runtime-config=experimental/v1=true \
>
> Reverted
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12217/files#r36504307>
> .
>



-- 
Clayton Coleman | Lead Engineer, OpenShift